src/lib/components/ToggleSwitch/style.js

```diff
@@ -38,7 +38,7 @@ export const LabelText = styled.span`
 
 export const InputHidden = styled.input`
   opacity: 0;
-  visibility: hidden;
+  display: hidden;
```

Comment:
The browser was unable to `autoFocus` a hidden Input that uses `visibility: hidden`.